_data/header-footer-content.yml

```diff
@@ -85,6 +85,10 @@ footer:
   subscribe:
     items:
     - 
+      link: "https://github.com/m-lab"
```

Comment:
The links seem to be concatenating to the site URL, so on the preview it looks like these won't work.